packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -265,6 +265,12 @@ interface DatabaseClusterBaseProps {
   * @default - if storageEncrypted is true then the default master key, no key otherwise
   */
   readonly storageEncryptionKey?: kms.IKey;
+  /**
```

Comment:
Let's add an empty line before this line.

Reply:
Added the empty line, also aligned the misaligned KMS comments